Source/WebCore/css/CSSCounterStyle.cpp

```diff
@@ -360,26 +368,26 @@ String CSSCounterStyle::initialRepresentation(int value) const
     return { };
 }
 
-String CSSCounterStyle::fallbackText(int value)
+String CSSCounterStyle::fallbackText(int value, TextDirection textDirection)
 {
     if (m_isFallingBack || !fallback().get()) {
         m_isFallingBack = false;
-        return CSSCounterStyleRegistry::decimalCounter()->text(value);
+        return CSSCounterStyleRegistry::decimalCounter()->text(value, textDirection);
     }
     m_isFallingBack = true;
-    auto fallbackText = fallback()->text(value);
+    auto fallbackText = fallback()->text(value, textDirection);
     m_isFallingBack = false;
     return fallbackText;
 }
 
-String CSSCounterStyle::text(int value)
+String CSSCounterStyle::text(int value, TextDirection textDirection)
```

Comment:
Feels weird to introduce code that isn't used yet... I would prefer having the RenderListMarker to be in the same patch.